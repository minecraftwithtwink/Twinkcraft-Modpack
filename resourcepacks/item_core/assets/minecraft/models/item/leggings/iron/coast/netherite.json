{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/leggings/base/iron/coast",
        "layer1": "minecraft:trims/items/leggings_trim/coast_netherite"
    }
}