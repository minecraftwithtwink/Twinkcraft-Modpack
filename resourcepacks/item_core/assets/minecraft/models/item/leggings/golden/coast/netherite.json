{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/leggings/base/golden/coast",
        "layer1": "minecraft:trims/items/leggings_trim/coast_netherite"
    }
}